test/errhandling/parallel/forall-iterator-throws-setup.chpl

```diff
@@ -26,9 +28,10 @@ iter myiter(nn: int, nt: int, param tag: iterKind) throws where tag == iterKind.
 }
 
 // coforall loop in leader should NOT get vector pragma
-iter myiter(nn: int, nt: int, param tag: iterKind) throws where tag == iterKind.leader {
+iter myiter(nn: int, nt: int, param tag: iterKind): nothing throws where tag == iterKind.leader {
   throw new owned StringError("Test error");
 
+  // This is not reachable.
   coforall i in 0..#nt {
     yield i*nn..#nn;
   }
```

Comment:
I suggest that these iterators should yield non-nothing types. The serial and the standalone should yield ints and the leader should yield `range`. Even though the yield statements will not be executed and the return type does not matter in this test, stylistically the return type should be the type of yielded expressions if there were executed.